app/utils/hyrax/test_data_seeders/user_seeder.rb

```diff
@@ -25,14 +25,14 @@ def generate_seeds(logger: Logger.new(STDOUT), allow_seeding_in_production: fals
         private
 
         def admin_role
-          unless defined? Role
+          unless ::User.reflect_on_association(:roles)
```

Comment:
`Role` seemed to not be loaded while running `rake db:seed`, so this instead checks if the user model has a roles association which `add_user` also depends on existing.